web/Views/Admin/PropertyList.cshtml

```diff
@@ -20,7 +20,7 @@
                     },
                     new { @class = "form-select col h-auto me-3", onchange = "displayIfSelectedValueIsTarget('Location', 'SortBy', 'postcode'); insertDefaultPostcodeIfNotSortByLocation(id)" })
                 <div id="postcode" class="col-auto" style="@(Model.SortBy != "Location" ? "display: none;" : "")">
-                    <input class="form-control col h-auto me-3" id="target" name="target" required placeholder="Enter postcode to sort by location" pattern="@Constants.PostcodeValidationRegex"/>
+                    <input class="form-control col h-auto me-3" id="target" name="target" required placeholder="Enter postcode to sort by location" pattern="@Constants.PostcodeValidationRegex" value="@(Model.SortBy == "Location" ? "" : "CB2 1LA")"/>

```

Comment:
Not anything to do with your code really, but we should probably change this placeholder postcode to be ChangeAhead's postcode rather than Softwire's

Reply:
That's a good point, I've changed it now!